backend/app_dev.yaml

```diff
@@ -0,0 +1,6 @@
+runtime: nodejs18
+env_variables:
+  #GCS_BUCKET: "atip-test-2"
```

Comment:
You can remove the two commented things, that looks like old stuff from when I was originally getting this to work in my project